app/views/home/_dashboard_paper.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if papers.any? %>
+  <% papers.each do |paper| %>
+  <div class="dashboard_paper">
+    <p class="title"><%= truncate(paper.title, length: 90) %></p>
+    <p class="meta"><a href="https://arxiv.org/abs/<%= paper.provider_id %>"><%= paper.typed_provider_id %></a> / <%= paper.authors.split(',').first %> / <%= paper.created_at.strftime('%d %B %Y') %></p>
+  </div>
+  <% end %>
+<% else %>
+  <p class="sorry">You don't have any papers in this category</a>
```

Comment:
Maybe we should just hide the category if they have no papers? I think it would be less confusing, for example most people will never see an editing paper